backend/tests/routers/test_aiod.py

```diff
@@ -0,0 +1,135 @@
+from unittest.mock import ANY
+
+import pytest
+
+from app.schemas.dataset import Dataset
+from app.schemas.ml_model import MLModel
+from app.schemas.platform import Platform
+from app.schemas.publication import Publication
+from app.services.aiod import AssetType
+
+
+@pytest.mark.parametrize(
+    "api_path, asset_type, asset_class",
+    [
+        ("/v1/assets/datasets", AssetType.DATASETS, Dataset),
+        ("/v1/assets/models", AssetType.ML_MODELS, MLModel),
+        ("/v1/assets/publications", AssetType.PUBLICATIONS, Publication),
+        ("/v1/assets/platforms", AssetType.PLATFORMS, Platform),
+    ],
+)
+@pytest.mark.asyncio
+async def test_api_get_assets(client, mocker, api_path, asset_type, asset_class):
```

Comment:
Also Pagination needs to be tested

Reply:
Do you mean just add some specific pagination in the `get_assets` and `search_assets` router tests and test whether the same pagination was used in the AIoD calls (I am currently using `ANY` in there)? That I can add for sure, but I am not sure what more to add here.